lib/group/group-router.js

```diff
@@ -1,56 +1,66 @@
 'use strict';
 
 var express = require('express'),
-  config = require('./config-group');
+  config = require('./config-group'),
+  shortid = require('shortid');
 
-function initRouter(mediator) {
+var GroupCloudDataTopics = require('fh-wfm-mediator/lib/topics');
+
+/**
+ * Function which handles the response according to the promise returned by the topic
+ *
+ * @param {Object} res
+ * @param {Promise} topicPromise Promise returned by the topic the request was sent to
+ */
+function handleResponse(res, topicPromise) {
+  topicPromise.then(function(data) {
+    res.status(200).json(data);
+  }).catch(function(err) {
+    console.log("[Error] Group topic request: ", err);
+    res.status(500).json({error: err.message});
+  });
+}
```

Comment:
I'm actually not sure about the catch suggestion. Any error thrown inside any promise chain is implicitly wrapped in a try/catch, so this error will just be propagated along as if this catch weren't even there. Unless Express is listening to the uncaught promise exception handler, it would never see that error. (I think there are Bluebird configurations for this.)